ports/kspalaiologos-bzip3/vcpkg.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "kspalaiologos-bzip3",
```

```suggestion
  "name": "bzip3",
```

I noticed that this package is `bzip3` in other package managers (https://repology.org/project/bzip3/versions). 

So even though it is unrelated to the owners of `bzip2` the community seems to have accepted the name, also the current prefix is hard to memorize. 

@toge sorry about the inconvenience, can you rename the port back to just `bzip3`.